### PR TITLE
Replace hostname and port with placeholders of callback URLs for console and myaccount

### PR DESCRIPTION
**Pull request by Thisara-Welmilla**
Issue:
- https://github.com/wso2/product-is/issues/10514

Replace hostname and port with placeholders of callback URLs for only console and myaccount when storing in databases and resolve the callback URL at retrieval.

### CODE REVIEW

File: components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth/dao/OAuthAppDAO.java [hunk 1]
```diff
@@ -414,7 +433,13 @@ public OAuthAppDO getAppInformation(String consumerKey, int tenantId) throws
                             authenticatedUser.setUserName(rSet.getString(2));
                             oauthApp.setApplicationName(rSet.getString(3));
                             oauthApp.setOauthVersion(rSet.getString(4));
+
```
Comment (chamathns) [inline components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth/dao/OAuthAppDAO.java:436]: redundant blank line

File: components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth/dao/OAuthAppDAO.java [hunk 2]
```diff
@@ -493,7 +521,13 @@ public OAuthAppDO getAppInformation(String consumerKey, AccessTokenDO accessToke
                         authenticatedUser.setUserName(rSet.getString(USERNAME));
                         oauthApp.setApplicationName(rSet.getString(APP_NAME));
                         oauthApp.setOauthVersion(rSet.getString(OAUTH_VERSION));
+
```
Comment (chamathns) [inline components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth/dao/OAuthAppDAO.java:524]: redundant blank line

File: components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth/dao/OAuthAppDAO.java [hunk 3]
```diff
@@ -559,7 +596,13 @@ public OAuthAppDO[] getAppsForConsumerKey(String consumerKey)
...
```
Comment (chamathns) [inline components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth/dao/OAuthAppDAO.java:599]: redundant blank line

File: components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth/dao/OAuthAppDAO.java [hunk 4]
```diff
@@ -634,7 +680,13 @@ public OAuthAppDO getAppInformationByAppName(String appName) throws
                                     (4));
                             oauthApp.setOauthConsumerKey(preprocessedClientId);
                             oauthApp.setOauthVersion(rSet.getString(5));
+
```
Comment (chamathns) [inline components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth/dao/OAuthAppDAO.java:683]: redundant blank line

**Review by chamathns — APPROVED** [review]

File: components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth/dao/OAuthAppDAO.java (hunk 1 above)
Comment (Thisara-Welmilla) [reply components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth/dao/OAuthAppDAO.java:436]: Added it to improve readability by separating the callback URL resolving logic.

File: components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth/dao/OAuthAppDAO.java (hunk 2 above)
Comment (Thisara-Welmilla) [reply components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth/dao/OAuthAppDAO.java:524]: Added it to improve readability by separating the callback URL resolving logic.

File: components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth/dao/OAuthAppDAO.java (hunk 4 above)
Comment (Thisara-Welmilla) [reply components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth/dao/OAuthAppDAO.java:683]: Added it to improve readability by separating the callback URL resolving logic.

File: components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth/dao/OAuthAppDAO.java (hunk 3 above)
Comment (Thisara-Welmilla) [reply components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth/dao/OAuthAppDAO.java:599]: Added it to improve readability by separating the callback URL resolving logic.

**Pull request merged**